naming/src/main/java/com/alibaba/nacos/naming/remote/rpc/filter/GrpcRequestFilter.java

```diff
@@ -0,0 +1,27 @@
+package com.alibaba.nacos.naming.remote.rpc.filter;
+
+import com.alibaba.nacos.api.exception.NacosException;
+import com.alibaba.nacos.api.remote.request.Request;
+import com.alibaba.nacos.api.remote.request.RequestMeta;
+import com.alibaba.nacos.api.remote.response.Response;
+import com.alibaba.nacos.core.remote.AbstractRequestFilter;
+import com.alibaba.nacos.naming.core.v2.upgrade.UpgradeJudgement;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GrpcRequestFilter extends AbstractRequestFilter {
+    
+    @Autowired
+    private UpgradeJudgement upgradeJudgement;
+    
+    @Override
+    protected Response filter(Request request, RequestMeta meta, Class handlerClazz) throws NacosException {
+        if (!upgradeJudgement.isUseGrpcFeatures()) {
+            Response response = getDefaultResponseInstance(handlerClazz);
```

Comment:
There need judge whether request is `naming` module, because the upgradeJudgement is only work for `naming`. `Config` module is not affected.

you can judgement request by `instance of `AbstractNamingRequest`

Reply:
Or judgement the handlerClazz also right.

Reply:
judgement about whether request instance of AbstractNamingRequest added.
Thanks for your review!